### PR TITLE
twilio doc

### PR DESCRIPTION
**Pull request by jpjoyal**
@spasiu @Mario54 @mspensieri @everyonewithmoreinspiration

### CODE REVIEW

File: source/includes/integrations/_twilio.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Twilio
+
+[Twilio](https://www.twilio.com/) allows you to get a phone number to send and receive SMS via their API. Our Twilio integration allows your users to send you text messages at your Twilio number for your to receice in any of our channel like Slack, Hipchat or HelpScout. Your replies are sent back as SMS to their phone.
```
Comment (dannytranlx) [inline source/includes/integrations/_twilio.md]: its a channelzz

File: source/includes/integrations/_twilio.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+### Configuring Twilio
+
+You'll need a Twilio account with a phone number configured on it. A trial account is ok to test but is only good to send SMS to the phone you used while signing up. Once equiped with your account, you need to copy your API credentials over Smooch.
+
+![Enter API credentials](/images/twilio.png)
+
+Once your Twilio account is authenticated, select the phone number you want to use and hit save. You should now be able to receive messages from your Twilio phone number and reply via any of your configured channel.
```
Comment (dannytranlx) [inline source/includes/integrations/_twilio.md]: chanelzzz

**Comment by alavers** [comment]
that screenshot... the left side is blurred out but the right side isn't. I vote blur them both, just so we don't look like dumbasses

File: source/includes/integrations/_twilio.md (hunk 1 above)
Comment (alavers) [inline source/includes/integrations/_twilio.md]: >[Twilio](https://www.twilio.com/) allows you to get a phone number to send and receive SMS messages via their API.

Comment (alavers) [inline source/includes/integrations/_twilio.md]: actually, the sentence is just a bit awkward. What about:

>[Twilio](https://www.twilio.com/) is an SMS messaging service.

Comment (alavers) [inline source/includes/integrations/_twilio.md]: >Your replies are sent back as an SMS to the user.

File: source/includes/integrations/_twilio.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+## Twilio
+
+[Twilio](https://www.twilio.com/) allows you to get a phone number to send and receive SMS via their API. Our Twilio integration allows your users to send you text messages at your Twilio number for your to receice in any of our channel like Slack, Hipchat or HelpScout. Your replies are sent back as SMS to their phone.
+
+### Configuring Twilio
+
+You'll need a Twilio account with a phone number configured on it. A trial account is ok to test but is only good to send SMS to the phone you used while signing up. Once equiped with your account, you need to copy your API credentials over Smooch.
```
Comment (alavers) [inline source/includes/integrations/_twilio.md]: Once you've set up your Twilio account, obtain your API credentials from the Twilio account settings page and enter them into the corresponding Smooch page.

**Comment by alavers** [comment]
rest looks :+1:

**Pull request merged**